core/templates/dev/head/exploration_editor/ExplorationEditor.js

```diff
@@ -641,9 +641,14 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
         siteAnalyticsService.registerSavePlayableExplorationEvent(
           explorationData.explorationId);
       }
-
       $scope.isSaveInProgress = true;
-      explorationData.save(changeList, commitMessage, function() {
+      explorationData.save(changeList, commitMessage, function(response) {
```

Comment:
I think moving the logging sounds reasonable; could you implement it in this PR?

It's also not very clear what `response` is in this function. Could you call it something more descriptive, and only pass in the keys of `data` that are needed?